build.gradle

```diff
@@ -22,7 +22,6 @@ repositories {
 }
 
 dependencies {
-    implementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
```

Comment:
> Deleted reference to Apache Commons IO because IntelliJ provides it as well. (We're already doing this with jsoup, so I figured why not)

But "doing this" with jsoup is the reason our current plugin version ain't gonna be compatible with IDEA 2022.3, right? So, that's one "why-not" I can think of.

What about "whys" then? Memory, I guess, is one. Do we know the sizes of these libs compared to the size of our plugin and IntelliJ? Are there other "whys"?